data/json/items/obsolete.json

```diff
@@ -517,5 +517,27 @@
         "volume": 1,
         "bashing": 4,
         "to_hit": -2
+    },
+    {
+        "id": "jar_V8",
```

Comment:
Needs extension to the migration code to do that. I suspect the loss of a charge isn't worth the complexity but if you disagree we could extend the migrations code.